network-api/networkapi/buyersguide/templates/fragments/chapter_table_of_contents.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="row">
+    <div class="pt-4 pt-md-5 {% if not page.notes %}pb-4 pb-md-5{% endif %} col-lg-8 offset-lg-2 justify-content-center">
+      <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center">
+        <div class="col-12">
+          {% for child in page.get_children %}
```

Comment:
we probably want `page.get_children.live`, so we don't surface drafts?

Reply:
Absolutely! Good catch!